pkg/tanka/load.go

```diff
@@ -94,6 +91,16 @@ func Eval(path string, opts Opts) (interface{}, error) {
 	return loader.Eval(path, LoaderOpts{opts.JsonnetOpts, opts.Name})
 }
 
+var registeredLoaders = []Loader{
+	&InlineLoader{},
+	&StaticLoader{},
+}
+
+// SetLoaders register custom loaders
+func SetLoaders(loaders ...Loader) {
+	registeredLoaders = loaders
+}
+
 // DetectLoader detects whether the environment is inline or static and picks
 // the approriate loader
 func DetectLoader(path string) (Loader, error) {
```

Comment:
```suggestion
// DetectLoader detects whether the environment is inline or static and picks
// the approriate loader
func DetectLoader(path string, loaders []Loaders) (Loader, error) {
```
I'm not sure this is an improvement, please discuss.

Reply:
The purpose to reuse functions in workflows.go (`Apply`, `Delete` ...)

if changed like this, we may upgrade workflow function like
`Apply(l Loader, opts tanka.ApplyOpts) error`
https://github.com/octohelm/tankax/blob/master/pkg/tankax/workflow.go#L15